### PR TITLE
feat(changelog): gpu-instances-added-new-gpu-l4-instances 2024-03-12

### PR DESCRIPTION
**Pull request by ofranc**
Reporter: Fabien DA SILVA

This PR is a new changelog contribution. It has been created automatically.

Make sure to review it carefully before merging it.

### CODE REVIEW

File: changelog/march2024/2024-03-12-gpu-instances-added-new-gpu-l4-instances.mdx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: New GPU L4 instances
+status: added
+author:
+    fullname: '#gpu-instances'
+    url: 'https://slack.scaleway.com'
+date: 2024-03-12
```
Comment (bene2k1) [inline changelog/march2024/2024-03-12-gpu-instances-added-new-gpu-l4-instances.mdx:7]:
```suggestion
date: 2024-03-15
```

File: changelog/march2024/2024-03-12-gpu-instances-added-new-gpu-l4-instances.mdx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+category: compute
+product: gpu-instances
+---
+
+We are excited to introduce our latest GPU Instances based on Nvidia L4 in the Paris (PAR2) and the Warsaw (WAW2) regions, a versatile and cost-effective GPU Instance
```
Comment (bene2k1) [inline changelog/march2024/2024-03-12-gpu-instances-added-new-gpu-l4-instances.mdx:12]:
```suggestion
We are excited to introduce our latest GPU Instances based on [Nvidia L4](https://www.nvidia.com/en-us/data-center/l4/) in the Paris (PAR2) and the Warsaw (WAW2) regions, a versatile and cost-effective GPU Instance.
```

File: changelog/march2024/2024-03-12-gpu-instances-added-new-gpu-l4-instances.mdx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+Key features include:
+- Nvidia L4 24GB (Ada Lovelace architecture)
+- 4th Tensor cores
```
Comment (bene2k1) [inline changelog/march2024/2024-03-12-gpu-instances-added-new-gpu-l4-instances.mdx:16]:
```suggestion
- 4th generation Tensor cores
```

not sure about this @fdasilva59

File: changelog/march2024/2024-03-12-gpu-instances-added-new-gpu-l4-instances.mdx [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+status: added
+author:
+    fullname: '#gpu-instances'
+    url: 'https://slack.scaleway.com'
+date: 2024-03-15
+category: compute
+product: gpu-instances
+---
+
+We are excited to introduce our latest GPU Instances based on [Nvidia L4](https://www.nvidia.com/en-us/data-center/l4/) in the Paris (PAR2) and the Warsaw (WAW2) regions, a versatile and cost-effective GPU Instance.
```
Comment (RoRoJ) [inline changelog/march2024/2024-03-12-gpu-instances-added-new-gpu-l4-instances.mdx:12]:
```suggestion
We are excited to introduce our latest versatile and cost-effective GPU Instances based on [Nvidia L4](https://www.nvidia.com/en-us/data-center/l4/) in the Paris (PAR2) and Warsaw (WAW2) regions.
```

File: changelog/march2024/2024-03-12-gpu-instances-added-new-gpu-l4-instances.mdx [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+Key features include:
+- Nvidia L4 24GB (Ada Lovelace architecture)
+- 4th generation Tensor cores
+- 4rd Generation RT cores (Graphics capability)
```
Comment (RoRoJ) [inline changelog/march2024/2024-03-12-gpu-instances-added-new-gpu-l4-instances.mdx:17]:
```suggestion
- 4rd generation RT cores (graphics capability)
```

File: changelog/march2024/2024-03-12-gpu-instances-added-new-gpu-l4-instances.mdx [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+- Available in multi-gpu 1, 2, 4 or 8 GPUs
```
Comment (RoRoJ) [inline changelog/march2024/2024-03-12-gpu-instances-added-new-gpu-l4-instances.mdx:18]:
```suggestion
- Available in multi-GPU 1, 2, 4 or 8 GPUs
```

**Review by nerda-codes — APPROVED** [review]

File: changelog/march2024/2024-03-12-gpu-instances-added-new-gpu-l4-instances.mdx [hunk 7]
```diff
@@ -0,0 +1,20 @@
+---
+title: New GPU L4 instances
```
Comment (nerda-codes) [inline changelog/march2024/2024-03-12-gpu-instances-added-new-gpu-l4-instances.mdx:2]:
```suggestion
title: New GPU L4 Instances
```

**Review by RoRoJ — APPROVED** [review]

File: changelog/march2024/2024-03-12-gpu-instances-added-new-gpu-l4-instances.mdx [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+    url: 'https://slack.scaleway.com'
+date: 2024-03-15
+category: compute
+product: gpu-instances
+---
+
+We are excited to introduce our latest versatile and cost-effective GPU Instances based on [Nvidia L4](https://www.nvidia.com/en-us/data-center/l4/) in the Paris (PAR2) and Warsaw (WAW2) regions.
+
+Key features include:
+- Nvidia L4 24GB (Ada Lovelace architecture)
```
Comment (jcirinosclwy) [inline changelog/march2024/2024-03-12-gpu-instances-added-new-gpu-l4-instances.mdx:15]:
```suggestion
- Nvidia L4 24 GB (Ada Lovelace architecture)
```

**Pull request merged**